Makefile

```diff
@@ -49,27 +45,34 @@ DOCKER_RUN_CMD  = docker run ${DOCKER_FLAGS} ${BUILD_TOOLS_DOCKER_IMAGE}
 .PHONY: default
 default: help
 
-## Run pre-commit hooks in build-tools docker container.
+# Not exposed as a callable target by `make help`, since this is a one-time shot to simplify the development of this module.
+.PHONY: template/adjust
+template/adjust: FILTER = -path ./.git -prune -a -type f -o -type f -not -name Makefile
+template/adjust:
+	@find . $(FILTER) -exec sed -i -e "s,terraform-module-template,$${PWD##*/},g" {} \;
+
```

Comment:
not needed as the comment says it's a one-time thing... ;) i assume just copied so let's keep it ;)

Reply:
yeah copy paste, didn't know whether we might need this later on. Let's keep it as it doesn't hurt :) I will try `git-xargs` later to get rid of it hehe